namespaces/live.cloud-platform.service.justice.gov.uk/laa-ccms-staging-api-preprod/01-rbac.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: laa-ccms-staging-api-dev-admin
```

Comment:
Should be name: laa-ccms-staging-api-preprod-admin

Reply:
updated